oneoffs/symmetry_analysis.py

```diff
@@ -0,0 +1,130 @@
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import argparse
+import json
+import os
+import sys; sys.path.insert(0, '.')
+
+import numpy as np
+import tensorflow as tf
+
+import dual_net
+import features
+import sgf_wrapper
+import symmetries
+
+
+def analyze_symmetries(sgf_file, load_file):
+    with open(sgf_file) as f:
+        sgf_contents = f.read()
+
+    iterator = sgf_wrapper.replay_sgf(sgf_contents)
+    net = dual_net.DualNetwork(load_file)
```

Comment:
consider passing an initialized DualNetwork instead of instantiating it within function; will speed up repeated analyses over many files

Reply:
Good idea--done.